scripts/interactions.js

```diff
@@ -308,7 +310,19 @@ damasflow_ondrop = function ( e )
 			}
 			else
 			{
-				if( confirm('Add ' + path + '?'))
+				if(oldPath===path){
+					var newWd=path.replace(/\/[^\/]*$/,"");
+					newWd=prompt("Ajouter � vos workdirs?",newWd);
+					if(newWd){
```

Comment:
I can't use this oneliner as I need the return value from the prompt for later use.